auction-deploy/tests/test_cli.py

```diff
@@ -94,3 +96,33 @@ def test_cli_auction_status(runner, deployed_auction_address):
     )
 
     assert result.exit_code == 0
+
+
+def test_cli_address_not_prefixed_hex(runner, deployed_auction_address):
+
+    hex_prefix_size = len("0x")
+
+    address = deployed_auction_address[hex_prefix_size:]
```

Comment:
Why do you remove the 0x?

Reply:
To see that it still works without, I don't care if we do not have it. I would rather be open when validating the address rather than strict.
If we enforce checksummed address then I'll remove this test.